src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [];
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (id) => ({
+  type: REMOVE_BOOK,
+  id,
+});
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [
+        ...state,
+        {
+          ...action.payload,
+          progress: {
+            currentChapter: 'NEW CHAPTER"',
+            completed: '0',
+          },
+        },
+      ];
+    case REMOVE_BOOK:
+      return state.filter((book) => book.id !== action.id);
+    default:
+      return state;
+  }
+};
+
+export default reducer;
```

Comment:
- [ ] Please implement the `remove` feature currently you have it as a placeholder because you can not remove a book that was not added

Reply:
ok, I fixed it.